e2e/rdctl.e2e.spec.ts

```diff
@@ -697,6 +729,88 @@ test.describe('Command server', () => {
         });
         expect(stderr).not.toContain('Usage:');
       });
+
+      test.describe('settings v5 migration', () => {
+        /** Note issue https://github.com/rancher-sandbox/rancher-desktop/issues/3829
```

Comment:
I believe normally `/**` is on a line by itself.

Reply:
ok